easybuild/easyconfigs/g/gocryptfs/gocryptfs-2.5.4.eb

```
@@ -0,0 +1,51 @@
+easyblock = 'ConfigureMake'
+
+name = 'gocryptfs'
+version = '2.5.4'
+
+homepage = 'https://nuetzlich.net/gocryptfs/'
+description = "An encrypted overlay filesystem written in Go. gocryptfs is built on top the excellent go-fuse FUSE library."
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/rfjakob/gocryptfs/releases/download/v%(version)s/']
+sources = ['gocryptfs_v%(version)s_src-deps.tar.gz']
+patches = ['gocryptfs-2.5.4_install.patch']
```

might as well use templates for the names
```suggestion
sources = ['%(name)s_v%(version)s_src-deps.tar.gz']
patches = ['%(name)s-2.5.4_install.patch']
```